firefox/src/background.ts

```diff
@@ -107,7 +107,12 @@ browser.webRequest.onHeadersReceived.addListener(
         setCookies({
           setCookieHeaders,
           requestUrl: info.url,
-          setCookie: (cookie) => browser.cookies.set(cookie),
+          setCookie: (cookie, domain) => {
+            if (!domain?.endsWith('.toddle.site')) {
```

Comment:
Why was this necessary?

Reply:
I was getting an error in the extension that is not possible to set a cookie for a domain that has no permission.